lib/bolt_server/transport_app.rb

```diff
@@ -30,6 +30,13 @@ def initialize(config)
       super(nil)
     end
 
+    def scrub_stack_trace(result)
+      if result[:result]['_error'] && result[:result]['_error']['details']['stack_trace']
```

Comment:
Seems like it is it safe to assume that every `result[:result]['_error'] has a `details` key. If a result does not have an `_error` key the `&&` condition will short circuit and avoid a `undefined method `[]' for nil:NilClass` problem for nested keys.

Reply:
`if result.dig(:result, '_error', 'details', 'stack_trace')`

Reply:
:man_facepalming: Of course. That was silly of me.